content/post/retour-sur-notre-rencontre-avec-la-cnil.en.md

```diff
@@ -1,40 +1,39 @@
+title: "[EN] Feedback on our meeting with the CNIL"
 date: 2018-02-28 19:56:18
 draft: false
 ---
 
-**Note: this post is only available in French for the time being.**
+On February 14, 2018, Exodus Privacy was invited by the CNIL (_"Commission Nationale de l'Informatique et des Libertés" in french, "National Commission for Computing and Liberties" in english_) to its premises to present itself and its activities.
 
-Le 14 février 2018, Exodus Privacy a été invitée par la CNIL (_Commission Nationale de l’Informatique et des Libertés_) dans ses locaux à se présenter et présenter ses activités.
+Were present for Exodus Privacy: [U+039b](https://esther.codes/about-me/) (president), [Lovis_IX](https://pleroma.tamanoir.foucry.net/users/LovisIX) (secretary) and [Maxime Auvy](https://open-freax.fr/) (member of the office). We were accompanied by a member of [La Quadrature du Net](https://www.laquadrature.net/en/).
 
-Étaient présents pour Exodus Privacy : [U+039b](https://esther.codes/about-me/) (présidente), [Lovis_IX](https://pleroma.tamanoir.foucry.net/users/LovisIX) (secrétaire) et [Maxime Auvy](https://open-freax.fr/ ) (membre du bureau). Nous étions accompagnés par un membre de [La Quadrature du Net](https://www.laquadrature.net/fr/).
+There were many participants on the CNIL side, around 15 people, from different departments. We were initially invited by Geoffrey Delcroix, from [LINC](https://linc.cnil.fr/fr/propos-de-linc) (_"Laboratoire d’Innovation Numérique de la CNIL" in french, "CNIL Digital Innovation Laboratory
+" in english _).
 
-Les participants coté CNIL étaient nombreux, environ 15 personnes, issus de différents services. Nous étions initialement invités par Geoffrey Delcroix, du [LINC](https://linc.cnil.fr/fr/propos-de-linc) (*Laboratoire d’Innovation Numérique de la CNIL*).
+Were represented in particular:
 
-Étaient représentés notamment :
+- the Compliance Department, which generally establishes the interpretation of the Data Protection Act, and the guidelines to be followed by organizations;
+- the Directorate for the Protection of Rights and Sanctions, and more particularly the Control Department, interested in using the εxodus tool as part of the preparation of its missions;
+- the Technological Expertise Department, with regard to more specifically the internal functioning of the tool.
 
-*  la Direction de la Conformité, qui établit globalement l’interprétation faite de la loi Informatique et Libertés, et les lignes directrices à suivre par les organismes ;
-*  la Direction de la Protection des Droits et des Sanctions, et plus particulièrement le Service des Contrôles, intéressé par l’utilisation de l'outil εxodus dans le cadre de la préparation de ses missions ;
-*  le Service de l’Expertise Technologique, pour ce qui touche plus spécifiquement au fonctionnement interne de l'outil.
+Several points were discussed, including:
 
-Plusieurs points ont été abordés, notamment :
+- the genesis of Exodus Privacy;
```

Comment:
Any particular reason why you replaced `*` by `-`?

Reply:
Sorry, Prettier have automatically replace `*` by `-` in `.md`. I fix this in next commit